docs/index.md

```diff
@@ -42,66 +42,62 @@ The [`jax-finufft`](https://github.com/dfm/jax-finufft) package is an optional d
 
 The following is a basic workflow to simulate an image.
 
-First, instantiate the scattering potential representation and its respective method for computing image projections.
+First, instantiate the spatial potential energy distribution representation and its respective method for computing image projections.
 
 ```python
 import jax
 import jax.numpy as jnp
 import cryojax.simulator as cxs
 from cryojax.data import read_array_with_spacing_from_mrc
 
-# Instantiate the scattering potential.
+# Instantiate the scattering potential
 filename = "example_scattering_potential.mrc"
 real_voxel_grid, voxel_size = read_array_with_spacing_from_mrc(filename)
 potential = cxs.FourierVoxelGridPotential.from_real_voxel_grid(real_voxel_grid, voxel_size)
-# ... now instantiate fourier slice extraction
-integrator = cxs.FourierSliceExtract(interpolation_order=1)
-```
-
-Here, the 3D scattering potential array is read from `filename`. Then, the abstraction of the scattering potential is then loaded in fourier-space into a `FourierVoxelGridPotential`, and the fourier-slice projection theorem is initialized with `FourierSliceExtract`. The scattering potential can be generated with an external program, such as the [cisTEM](https://github.com/timothygrant80/cisTEM) simulate tool.
-
-We can now instantiate the representation of a biological specimen, which also includes a pose.
-
-```python
-# First instantiate the pose. Angles are given in degrees
+# ... now, instantiate the pose. Angles are given in degrees
 pose = cxs.EulerAnglePose(
     offset_x_in_angstroms=5.0,
     offset_y_in_angstroms=-3.0,
     view_phi=20.0,
     view_theta=80.0,
     view_psi=-10.0,
 )
-# ... now, build the biological specimen
-specimen = cxs.Specimen(potential, integrator, pose)
+# ... now, build the ensemble. In this case, the ensemble is just a single structure
+structural_ensemble = cxs.SingleStructureEnsemble(potential, pose)
 ```
 
-Next, build the model for the electron microscope. Here, we simply include a model for the CTF in the weak-phase approximation (linear image formation theory).
+Here, the 3D scattering potential array is read from `filename`. Then, the abstraction of the scattering potential is then loaded in fourier-space into a `FourierVoxelGridPotential`. The scattering potential can be generated with an external program, such as the [cisTEM](https://github.com/timothygrant80/cisTEM) simulate tool. Then, the representation of a biological specimen is instantiated, which also includes a pose and conformational heterogeneity. Here, the `SingleStructureEnsemble` class takes a pose but has no heterogeneity.
+
+Next, build the *scattering theory*. The simplest `scattering_theory` is the `LinearScatteringTheory`. This represents the usual image formation pipeline in cryo-EM, which forms images by computing projections of the potential and convolving the result with a contrast transfer function.
```

Comment:
I would put some references here to the weak phase object approximation, and projection approximation.

Reply:
Maybe we should have a references section in the README?